web/css/joindin.css

```diff
@@ -624,6 +624,19 @@ a.talk.star {
     color: #777;
 }
 
+.event-attendees img {
+    max-width:100%
+}
+
+.event-attendees .row .col {
+    text-align:center;
+}
+
+.event-attendees .row .col .attendeeName {
+    height:40px;
```

Comment:
Ah you're right! I forgot to check that.
I've played with that a little more, I was just bothered by the fact that there's no spacing between the name and the avatar. Can I suggest this approach then?
```
    height: 42px;
    padding-top: 2px;
```